docs/sources/datasources/elasticsearch/query-editor/index.md

```diff
@@ -15,59 +15,164 @@ labels:
+Configure the following options for the **histogram** bucket aggregation option:
 
-{{< figure src="/static/img/docs/elasticsearch/pipeline-aggregation-editor-7-4.png" max-width="500px" class="docs-image--no-shadow" caption="Pipeline aggregation editor" >}}
+  - **Interval** - Group by a type of interval. There are option to choose from the dropdown menu to select seconds, minutes, hours or day. You can also add a custom interval such as `30d` (30 days). `Auto` is the default option.
+  - **Min doc count** - The minimum amount of data to include in your query. The default is `0`
 
-## Create a query
+The **nested** group by option is currently experimental, you can select a field and then settings specific to that field. 
 
-Write the query using a custom JSON string, with the field mapped as a [keyword](https://www.elastic.co/guide/en/elasticsearch/reference/current/keyword.html#keyword) in the Elasticsearch index mapping.
+Click the **+ sign** to add multiple group by options. The data will grouped in order (first by, then by).
 
-If the query is [multi-field](https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html) with both a `text` and `keyword` type, use `"field":"fieldname.keyword"` (sometimes `fieldname.raw`) to specify the keyword field in your query.
+{{< figure src="/static/img/docs/elasticsearch/group-by-then-by-10.2.png" max-width="850px" class="docs-image--no-shadow" caption="Group by options" >}}
 
-| Query                                                               | Description                                                                                                                                                                   |
-| ------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `{"find": "fields", "type": "keyword"}`                             | Returns a list of field names with the index type `keyword`.                                                                                                                  |
-| `{"find": "terms", "field": "hostname.keyword", "size": 1000}`      | Returns a list of values for a keyword using term aggregation. Query will use current dashboard time range as time range query.                                               |
-| `{"find": "terms", "field": "hostname", "query": '<Lucene query>'}` | Returns a list of values for a keyword field using term aggregation and a specified Lucene query filter. Query will use current dashboard time range as time range for query. |
+### Logs query type
+
+Logs queries analyze Elasticsearch log data. You can configure the following options:
+
+- **Logs Options/Limit** - Limits the number of logs to analyze. The default is `500`.
+
+#### Logs volume panel
```

Comment:
I still think that this whole section `Logs volume panel` and `Logs panel` should not be here, cause it is not related to query editor. We don't do it for metrics or raw query results. And I think it could be confusing for everyone who uses Elasticsearch in dashboard/alerting, as `Logs volume panel` and `Logs panel` are just visualisations used in Explore.

Reply:
Because it's there when you open the query editor, we should document what it is.  I personally am not sure query editor docs should be under data sources, as it's basically clicking on Explore, and Explore is documented elsewhere.

Reply:
But if you open Elasticsearch editor in Dashboard, it is not going to be there - see https://play.grafana.org/d/play-elastic-web-logs/elasticsearch-8-web-logs?orgId=1&editPanel=1
[<img width="1484" alt="image" src="https://github.com/grafana/grafana/assets/30407135/6153c70a-47a9-4efb-bd8f-1d8ff6ea5aa8">](https://play.grafana.org/d/play-elastic-web-logs/elasticsearch-8-web-logs?orgId=1&editPanel=2)